### PR TITLE
Add Discussions to triage

### PR DESCRIPTION
**Pull request by Fryguy**

### CODE REVIEW

File: triage_process.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ The ManageIQ core team holds regular meetings to triage issues and pull requests
   * [Unassigned PRs][]
   * [PRs without scope label][]
   * [Stale PRs][]
+* Discussions
+  * [Open discussions][]
```
Comment (jrafanie) [inline triage_process.md:23]: Should we have a link for Unanswered Q&A also?  I think that would follow the pattern above where we assume triage will answer questions and close open discussions either by creating issues from them or asking for information that we'll review at the next triage.  At some point, if any question goes without reply, we'll close them.

Comment (jrafanie) [reply triage_process.md:23]: or we can imply that open discussions will include both discussions and the special case q and a.

Comment (Fryguy) [reply triage_process.md:23]: yeah I was thinking the latter at least for this initial pass.  If we feel breaking it up into separate line items is helpful we can do that in a follow up.

Comment (jrafanie) [reply triage_process.md:23]: yeah, let's get this in and dog-food it.

**Pull request merged**